array/homework3.php

```diff
@@ -0,0 +1,52 @@
+<?php
+$multiArray =array(
+    array(5, 12, 17, 9, 13),
+    array(13, 4, 8, 14,1),
+    array(9, 5, 3, 17,21)
+);
+
+$sum = 0;
+$avg = 0;
+$min = 100;
+$max = 0;
+
+for ($i=0; $i<sizeof($multiArray); ++$i) {
+    $subSum = 0;
+    $subMin = 100;
+    $subMax = 0;
```

Comment:
You don't need to use sub variables here, you can calculate or check it directly like the hint.

Reply:
I revised sentences into ones using `foreach` & without sub-variables.